frontend/src/router.js

```diff
@@ -12,86 +12,107 @@ export default new Router({
+        default: () => import(/* webpackChunkName: "about" */ './views/Home'),
+        bottombar: () => import(/* webpackChunkName: "navigation" */ './views/BottomBar')
+      },
+      beforeEnter: requireAuth
+    },
+    {
+      path: '/profile',
+      name: 'profile',
+      components: {
+        topbar: () => import(/* webpackChunkName: "navigation" */ './views/TopBar'),
+        default: () => import(/* webpackChunkName: "about" */ './views/Profile'),
+        bottombar: () => import(/* webpackChunkName: "navigation" */ './views/BottomBar')
+      },
       beforeEnter: requireAuth
     },
     {
       path: '/camps',
       name: 'camps',
       components: {
-        default: () => import(/* webpackChunkName: "camps" */ './views/Camps.vue')
+        topbar: () => import(/* webpackChunkName: "navigation" */ './views/TopBar'),
+        default: () => import(/* webpackChunkName: "camps" */ './views/Camps'),
+        bottombar: () => import(/* webpackChunkName: "navigation" */ './views/BottomBar')
       },
       beforeEnter: requireAuth
     },
     {
       path: '/camps/:campId/:campTitle?',
       components: {
-        default: () => import(/* webpackChunkName: "camp" */ './views/Camp.vue'),
-        aside: () => import(/* webpackChunkName: "camps" */ './views/Camps.vue')
+        topbar: () => import(/* webpackChunkName: "navigation" */ './views/camp/TopBar'),
+        default: () => import(/* webpackChunkName: "camp" */ './views/camp/Camp'),
+        aside: () => import(/* webpackChunkName: "periods" */ './views/camp/Periods'),
+        bottombar: () => import(/* webpackChunkName: "navigation" */ './views/camp/BottomBar')
       },
       beforeEnter: requireAuth,
       props: {
-        default: route => ({ camp: campFromRoute(route) })
+        default: route => ({ camp: campFromRoute(route) }),
+        aside: route => ({ camp: campFromRoute(route) })
       },
       children: [
         {
           path: 'collaborators',
           name: 'camp/collaborators',
-          component: () => import(/* webpackChunkName: "campCollaborators" */ './components/camp/Collaborators.vue'),
-          meta: { layout: 'camp' }
-        },
-        {
-          path: 'periods',
-          name: 'camp/periods',
-          component: () => import(/* webpackChunkName: "campPeriods" */ './components/camp/Periods.vue'),
-          meta: { layout: 'camp' }
+          component: () => import(/* webpackChunkName: "campCollaborators" */ './views/camp/Collaborators.vue')
         },
         {
-          path: 'picasso',
-          name: 'camp/picasso',
-          component: () => import(/* webpackChunkName: "campPicasso" */ './components/camp/CampPicasso.vue'),
-          meta: { layout: 'camp' }
+          path: 'overview',
+          name: 'camp/overview',
+          component: () => import(/* webpackChunkName: "campPicasso" */ './views/camp/CampOverview'),
+          beforeEnter: parseBooleanInQuery
         },
         {
-          path: '',
```

Comment:
Du hast die Route mit dem leeren Pfad entfernt. Was jetzt also nicht mehr geht ist /camps/eafb123/So-La-2020. Finde ich vertretbar, auch wenn ich lieber /camps/eafb123/So-La-2020 als /camps/eafb123/So-La-2020/overview hätte.

Reply:
Ich habe nicht herausgefunden, wie ich diesen Pfad umleiten kann, so dass aber nicht bei einem Link mit Query (?list=false) noch zusätzlich ein /camps/eafb123/So-La-2020/overview/**overview** hinzugefügt.

Reply:
```js

      children: [
        {
          path: 'collaborators',
          name: 'camp/collaborators',
          component: () => import(/* webpackChunkName: "campCollaborators" */ './views/camp/Collaborators.vue')
        },
        {
          path: 'admin',
          name: 'camp/admin',
          component: () => import(/* webpackChunkName: "campAdmin" */ './views/camp/Admin')
        },
        {
          path: '',
          name: 'camp/overview',
          alias: 'overview',
          component: () => import(/* webpackChunkName: "campPicasso" */ './views/camp/CampOverview'),
          beforeEnter: parseBooleanInQuery
        }
      ]
```
So geht alles was ich gerade teste, übersehe ich etwas? Den Alias kannst du theoretisch auch weglassen, weil der eh nie in den verlinkten URLs stehen wird (ausser man schreibts von Hand dazu).
Wichtig ist aber dass der Eintrag mit leerem `path` am Ende der Liste steht. Nur so wird /camps/4636d1c8/admin korrekt verarbeitet.